### PR TITLE
Create CLI command to describe shard by id

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**
Created an admin CLI command that allows to describe ShardInfo providing an ID of the shard

**Why?**
Allows to discover and get information on concrete ShardInfo 's

**How did you test it?**
Executed the command providing few IDs

**Potential risks**
No risks

### CODE REVIEW

**Comment by feedmeapples** [comment]
This PR depends on the work from this one https://github.com/temporalio/temporal/pull/369
I will add reviewers after the former is merged

File: tools/cli/admin.go [hunk 1]
```diff
@@ -811,9 +831,15 @@ func newDBCommands() []cli.Command {
...
+		cli.StringFlag{
+			Name:  FlagDBEngine,
+			Value: "cassandra",
+			Usage: "Type of the DB engine to use (cassandra, mysql..)",
+		},
 		cli.StringFlag{
 			Name:  FlagDBAddress,
-			Usage: "persistence address(right now only cassandra is supported)",
+			Value: "127.0.0.1",
+			Usage: "persistence address(right now only cassandra is fully supported)",
```
Comment (alexshtin) [inline tools/cli/admin.go:842]:
```suggestion
			Usage: "persistence address (right now only cassandra is fully supported)",
```

File: tools/cli/admin.go [hunk 2]
```diff
@@ -117,6 +117,26 @@ func newAdminWorkflowCommands() []cli.Command {
 
 func newAdminShardManagementCommands() []cli.Command {
 	return []cli.Command{
+		{
+			Name:    "describe",
+			Aliases: []string{"d"},
+			Usage:   "Describe shard by id",
```
Comment (alexshtin) [inline tools/cli/admin.go:123]:
```suggestion
			Usage:   "Describe shard by Id",
```

File: tools/cli/admin.go [hunk 3]
```diff
@@ -117,6 +117,26 @@ func newAdminWorkflowCommands() []cli.Command {
...
+			Flags: append(
+				getDBFlags(),
+				cli.IntFlag{
+					Name:  FlagShardID,
+					Usage: "The ID of the shard to describe",
```
Comment (alexshtin) [inline tools/cli/admin.go:128]:
```suggestion
					Usage: "The Id of the shard to describe",
```

**Pull request merged**